targets/TARGET_STM/device.h

```diff
@@ -37,4 +37,7 @@
 
 #include "objects.h"
 
+#include "us_ticker_defines.h"
+#include "lp_ticker_defines.h"
```

Comment:
Could you make this include, or the contents of the header file, conditional on `DEVICE_LPTICKER` or whatever it is?

I'd rather the macros were left undefined in that case. There may not be any problem in the Mbed OS tree, but it's possible someone might assume LPTICKER is available if `LP_TICKER_PERIOD_NUM` is defined.

Reply:
@kjbracey-arm 
Done